### PR TITLE
Backlog 16359 scroll table cells

### PR DESCRIPTION
**Pull request by AKarmanov**
## JIRA

https://jira.jahia.org/browse/BACKLOG-16359

## Description

Allow cell to scroll when it contains overflowing content

### CODE REVIEW

File: src/components/Table/table-cells/TableBodyCell.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ export const TableBodyCell: React.FC<TableCellProps> = ({
     // TODO: When no iconStart is set we should just render children to avoid too nested HTML elements
     // TODO: Add iconEnd somewhere or remove it from the props, it seems we never use it
     const renderCellContent = () => (
-        <IconTextIcon component="div" iconStart={iconStart}>
+        <IconTextIcon component="div" iconStart={iconStart} typographyProps={{className: clsx('moonstone-tableCellContent', 'flexFluid')}}>
```
Comment (ffffffelix) [inline src/components/Table/table-cells/TableBodyCell.tsx:31]: I think it's better to add a prop to manage if the cell is scrollable or not. The default behavior must still be to truncate the text when it's too long.
In that case, we can rename the CSS class by something more descriptive like `moonstone-tableBodyCell_scrollable`.

File: src/components/Table/table-cells/TableCell.scss [hunk 2]
```diff
@@ -1,5 +1,23 @@
 @import '../../../utils/index.scss';
 
+.moonstone-tableCellComponent {
+    overflow-y: auto;
+
+    &::-webkit-scrollbar {
+        height: 2px;
```
Comment (ffffffelix) [inline src/components/Table/table-cells/TableCell.scss:7]: As the height only applies to WebKit browser the behavior is a little weird with other browsers like Firefox. I don't know how and if we can manage that ...

Comment (AKarmanov) [reply src/components/Table/table-cells/TableCell.scss]: I increased it to 4px actually, we can have it it doesn't distort anything in ff. I also added a similar prop for ff but it is still quite wide, covers content although it is semi transparent and you can still see a thing or two.

File: src/components/Table/table-cells/TableBodyCell.tsx [hunk 3]
```diff
@@ -70,6 +71,7 @@ export const TableBodyCell: React.FC<TableCellProps> = ({
     return (
         <Typography
             className={clsx(
+                'moonstone-tableCellComponent',
```
[the remark this answers is not in the document]
Comment (AKarmanov) [reply src/components/Table/table-cells/TableBodyCell.tsx]: 👍

File: src/components/Table/table-cells/TableBodyCell.tsx (hunk 1 above)
Comment (AKarmanov) [reply src/components/Table/table-cells/TableBodyCell.tsx]: 👍

**Review by ffffffelix — APPROVED** [review]

**Pull request merged**